Orange/canvas/report/report.py

```diff
@@ -162,8 +163,7 @@ def report_plot(self, name=None, plot=None):
             self.report_html += get_html_img(plot.scene())
 
     # noinspection PyBroadException
-    def report_table(self, name, table=None, header_rows=0, header_columns=0,
-                     num_format=None):
+    def report_table(self, name, table=None, header_rows=0, header_columns=0, confusion_rows=0, num_format=None):
```

Comment:
I just used the extra parameter to bold the last row of the confusion matrix. If I remove it, everything will be fine except the last row will **NOT** be bold.
So it fine in that case?